merkletree/tb.go

```diff
@@ -25,3 +29,15 @@ func (tb *TemporaryBinding) Serialize(strSig []byte) []byte {
 	tbBytes = append(tbBytes, tb.Value...)
 	return tbBytes
 }
+
+// Verify verifies the issued temporary binding whether
+// it was inserted as promised or not. This method should be
+// called right after the directory has been updated, i.e., at the next epoch.
+func (tb *TemporaryBinding) Verify(ap *AuthenticationPath) bool {
```

Comment:
The comment here suggests that the TB struct should contain an `epoch` field (the one it was handed out in) so that the keyserver can't delay adding the binding to the tree.